lib/open_food_network/sales_tax_report.rb

```diff
@@ -9,9 +9,16 @@ def initialize(user, params)
     end
 
     def header
-      ["Order number", "Date", "Items", "Items total (#{currency_symbol})", "Taxable Items Total (#{currency_symbol})",
-        "Sales Tax (#{currency_symbol})", "Delivery Charge (#{currency_symbol})", "Tax on Delivery (#{currency_symbol})", "Tax on Fees (#{currency_symbol})",
-        "Total Tax (#{currency_symbol})", "Customer", "Distributor"]
+      case params[:report_type]
+      when "tax_types"
+        ["Order number", "Date", "Items", "Items total (#{currency_symbol})", "Taxable Items Total (#{currency_symbol})",
+          "Sales Tax (#{currency_symbol})", "Delivery Charge (#{currency_symbol})", "Tax on Delivery (#{currency_symbol})", "Tax on Fees (#{currency_symbol})",
+          "Total Tax (#{currency_symbol})", "Customer", "Distributor"]
+      else
+        ["Order number", "Total excl. VAT"] +
+          relevant_rates.map { |rate| "%s (%.1f%%)" % [rate.name, rate.amount.to_f * 100] } +
```

Comment:
This is cool, but I think that displaying it in this way will be slightly odd for some countries. For example: in Australia, we only have one rate of tax, and it either applies or it doesn't, so there will be no difference between this per-rate total column and the `Total VAT` column. Not sure what to do about this - we will test it in the browser and then work out what we can do.